apps/console/src/features/users/components/guests/pages/invite-parent-org-user.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { Field, FormValue, Forms } from "@wso2is/forms";
+import {
+    Hint,
+    Message
+} from "@wso2is/react-components";
+import React, { ReactElement } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Divider, Grid } from "semantic-ui-react";
+import { AppState } from "../../../../core";
+import { UserInviteInterface } from "../api/invite";
+
+/**
+ * Proptypes for the invite parent org user component.
+ */
+interface InviteParentOrgUserProps extends IdentifiableComponentInterface {
+    triggerSubmit: boolean;
```

Comment:
Lets document these props.

```
```suggestion
   /**
     * Triggers a submit ...
    **/ 
    triggerSubmit: boolean;
```